### DISCUSSION TITLE
Alexa/Google Home

### DISCUSSION THREAD

**Discussion by pirakleous** [hass-emulated-hue/core#90] [Q&A] [answered]
Hello,

Can i control the devices with alexa/google home?

I am not able to find any discussion about that

Thanks

**Comment by alexyao2015** [comment, marked as answer]
Through Home Assistant normally. There is documentation on how to do so in Home Assistant's documentation

**Comment by pirakleous** [comment]
I was looking if i can link the device on philips hue website so i can connect the bridge to google/alexa. I connected the bridge in Home Assistant but i cannot add it on google home since the bridge is not connected with the philips account

**Reply by alexyao2015** [reply]
No that will never work. Why would you connect the bridge to Home Assistant? All lights are already in Home Assistant. Use Home Assistant to configure cloud services.
